.github/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 <h3>New features since last release</h3>
 
+* `Connection` objects now send versioned requests to the platform API.
+  [(#512)](https://github.com/XanaduAI/strawberryfields/pull/512)
```

Comment:
I would move this down to the 'improvements' section, since it is less visible to users!

Reply:
Also, don't forget to add yourself to the contributors section!